gateway-ha/src/main/java/io/trino/gateway/ha/router/HaQueryHistoryManager.java

```diff
@@ -29,14 +29,10 @@ public void submitQueryDetail(QueryDetail queryDetail) {
   public List<QueryDetail> fetchQueryHistory(Optional<String> user) {
     try {
       connectionManager.open();
-      String sql = "select * from query_history";
       if (user.isPresent()) {
-        sql += " where user_name = '" + user.get() + "'";
+        return QueryHistory.upcast(QueryHistory.where("user_name = '" + user.get() + "'").limit(2000).orderBy("created desc"));
```

Comment:
use `QueryHistory.userName` instead of `"user_name"` and `QueryHistory.created` instead of `"created"`

Reply:
It appears all the attributes of the activejdbc models are private. Should we add getters for them? Not sure what the typical activejdbc pattern is for this.